code/FulpstationCode/fulpops.dm

```diff
@@ -0,0 +1,32 @@
+/datum/outfit/syndicate/plasmaman
+	name = "Syndicate Operative Plasmaman - Basic"
+
+	head = /obj/item/clothing/head/helmet/space/plasmaman/robotics/syndicate
+	uniform = /obj/item/clothing/under/plasmaman/security/syndicate
+	r_hand= /obj/item/tank/internals/plasmaman/belt/full
```

Comment:
You've made sure this isnt replacing or being replaced by anything right?

If it is, add whatever it's having issues with to the hand slot.

Reply:
Those don't replace anything as the outfit is loaded instead of the normal syndie one. Their equivalent is just normal syndie gear and empty hands.